resources/views/listing/partials/filter/boolean.blade.php

```diff
@@ -19,7 +19,10 @@ class="relative pb-4"
                         v-bind:checked="value[item.key]"
                         v-on:change="handleChange(item.key)"
                     >
-                        <div class="font-sans font-medium text-sm items-center flex text-inactive" :class="{'text-neutral': value[item.key] == true}">
+                        <div
+                            class="font-sans font-medium text-sm items-center flex"
+                            :class="value[item.key] ? 'text' : 'text-muted'"
```

Comment:
i see border-default but here i still see "text" do we use "-default" everywhere or do we use "border", "bg", "text"?

Reply:
Just `border`, `bg` and `text` where ever you can. The `border-default` is only used to overwrite a border color (like with Tailwind Forms as mentioned in the PR description)